deploy/helm/kuberhealthy/values.yaml

```
@@ -138,6 +138,8 @@ check:
       tag: v1.2.2
     extraEnvs:
     nodeSelector: {}
+    cluster:
+      enabled: false
```

This seems a little vague.  Could we add something like `allNamespaces` here?

```suggestion
    allNamespaces: true
```